Frameworks/UIKit/UIColor.mm

```diff
@@ -226,6 +226,8 @@ @implementation UIColor {
     UIImage* _image;
     id _pattern;
     __CGColorQuad _components;
+    // Note: This should be part of the CGColor struct
+    woc::StrongCF<CGColorSpaceRef> _colorSpace;
```

Comment:
Talked to raj regarding this, ideally, the whole thing should be fixed!
but given the load this iteration and the re-write is coming up, this should be sufficient. 
as much as I don't like doing this, until the re-write happens this would fix up some stuff for us.